.github/workflows/bootc-e2e-nightly-windows.yaml

```diff
@@ -0,0 +1,236 @@
+name: Podman Desktop BootC E2E Nightly
+
+on:
+  schedule:
+    - cron:  '0 4 * * *'
+  workflow_dispatch:
+    inputs:
+      fork:
+        default: 'containers'
+        description: 'Podman Desktop repo fork'
+        type: string
+        required: true
+      branch:
+        default: 'main'
+        description: 'Podman Desktop repo branch'
+        type: string
+        required: true
+      ext_repo_options:
+        default: 'REPO=podman-desktop-extension-bootc,FORK=containers,BRANCH=main'
+        description: 'Podman Desktop Extension repo, fork and branch'
+        type: string
+        required: true
+      ext_tests:
+        default: '1'
+        description: 'Run E2E tests from extension'
+        type: string
+        required: true
+      npm_target:
+        default: 'test:e2e'
+        description: 'npm target to run tests'
+        type: string
+        required: true
+      podman_remote_url:
+        default: 'https://github.com/containers/podman/releases/download/v5.0.2/podman-remote-release-windows_amd64.zip'
+        description: 'podman remote zip'
+        type: string
+        required: true
+      podman_version:
+        default: '5.0.2'
+        description: 'Podman folder version in archive'
+        type: 'string'
+        required: true
+      podman_options:
+        default: 'INIT=1,START=1,ROOTFUL=1,NETWORKING=0'
+        description: 'Podman machine configuration options, no spaces'
+        type: 'string'
+        required: true
+      env_vars:
+        default: 'TEST_PODMAN_MACHINE=true'
+        description: 'Env. Variables passed into target machine, ie: VAR1=xxx,VAR2=true,VAR3=15,VAR4="Pass me along"'
+        type: 'string'
+        required: true
+
+jobs:
+  windows:
+    name: ${{ matrix.windows-version }} - Debug
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      checks: write
+    env:
+      QENVS_VERSION: v0.6.3
+    strategy:
+      fail-fast: false
+      matrix:
+        windows-version: ['10','11']
+        windows-featurepack: ['22h2-ent', '23h2-ent']
+        exclude:
+        - windows-version: '10'
+          windows-featurepack: '23h2-ent'
+        - windows-version: '11'
+          windows-featurepack: '22h2-ent'
+
+    steps:
+    - name: Set the default env. variables
+      env:
+        DEFAULT_FORK: 'containers'
+        DEFAULT_BRANCH: 'main'
+        DEFAULT_EXT_TESTS: '1'
+        DEFAULT_NPM_TARGET: 'test:e2e'
+        DEFAULT_ENV_VARS: 'TEST_PODMAN_MACHINE=true'
+        DEFAULT_PODMAN_OPTIONS: 'INIT=1,START=1,ROOTFUL=1,NETWORKING=0'
+        DEFAULT_EXT_REPO_OPTIONS: 'REPO=podman-desktop-extension-bootc,FORK=containers,BRANCH=main'
+        DEFAULT_VERSION: '5.0.2'
+        DEFAULT_URL: 'https://github.com/containers/podman/releases/download/v5.0.2/podman-remote-release-windows_amd64.zip'
```

Comment:
you need to update all extensions when there is a new version of podman included in Podman Desktop ?

shouldn't be part of some data grabbed from podman desktop

Reply:
That is a good idea.